### PR TITLE
feat(tactic/change): allow `change` to be used to rename bound variables

### PR DESCRIPTION
**Pull request by cipher1024**
# Pull Request Description

Ensure you have read the contribution guide before filling in a description of the
pull request, regardless of whether it is complete or a work in progress.
All Pull Requests should include test case(s) which demonstrates the intended
behavior of a feature, or a regression test demonstrating that the fix resolves
the issue.

### CODE REVIEW

File: tests/lean/change_tac.lean [hunk 1]
```diff
@@ -45,3 +45,14 @@ example (h : 1 + 2 = 1 + 2 + 1) : 1 + 2 = 3 :=
 begin
   change 1 + 2 with 4 at *, -- ERROR
 end
+
+example : true :=
```
Comment (robertylewis) [inline tests/lean/change_tac.lean]: Does this need a corresponding file with the expected output? I'm not sure what the test is right now, it should pass with any output from trace, right?

Comment (gebner) [reply tests/lean/change_tac.lean]: There's already a file with the expected output, it just needs to be updated.

@cipher1024 You can run the tests locally on your machine by executing `ctest` in the build directory.  (Or `ctest -R leantest` if you only want to run the `tests/lean` tests.)  You can also run a single test using the `test_single.sh` script in the `tests/lean` directory.

Comment (cipher1024) [reply tests/lean/change_tac.lean]: Sorry, I forgot it wasn't it my normal `make` command for Lean 3. The fix is coming

**Comment by bryangingechen** [comment]
Don't mind me, just commenting here so that issue #95 gets linked.

**Pull request merged**